### PR TITLE
Cybereason malop processes bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35374

## Description
The command used to work in the past, but after some code changes, it stopped working. The reason was that by a mistake some code was moved under an unwanted if condition. Attached the previous version of the code (the version which was working well).

## Screenshots
Previous working version of the code:
image

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cybereason/ReleaseNotes/1_0_7.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cybereason
+- Fixed an issue where the command ***cybereason-malop-processes*** used to work in previous version but stopped working.
```
Comment (bakatzir) [inline Packs/Cybereason/ReleaseNotes/1_0_7.md:4]:
```suggestion
- Fixed an issue where the ***cybereason-malop-processes*** command was not showing the proper outputs.
```

**Review by bakatzir — CHANGES_REQUESTED** [review]
I suggest adding a UT to the `cybereason-malop-processes` cmd, testing a proper context outputs creation.

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 2]
```diff
@@ -673,13 +673,13 @@ def malop_processes_command():
             if not wanted_machine:
                 continue
 
-            output = {}
-            for info in PROCESS_INFO:
-                if item.get('type', '') == 'filterData':
-                    output[info['header']] = dict_safe_get(item, ['filterData', 'groupByValue'])
+        output = {}
+        for info in PROCESS_INFO:
+            if info.get('type', '') == 'filterData':
```
Comment (reutshal) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py:678]: Why did you change this from item to info?
Won't it change the behavior here?

File: Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py [hunk 3]
```diff
@@ -307,3 +314,20 @@ def test_query_file(mocker):
...
                                      'val.SHA1===obj.SHA1)'][0]['Machine'] == 'desktop-p0m5vad'
+
+
+def test_malop_processes_command(mocker):
+    mocker.patch.object(demisto, 'params', return_value=params)
+    mocker.patch.object(demisto, 'args', return_value={"malopGuids": ["11.-6236127207710541535"]})
+    raw_response = util_load_json('test_files/malop_processes_raw_response.json')
+    mocker.patch('Cybereason.malop_processes', return_value=raw_response)
+    mocker.patch.object(demisto, 'results')
+    import Cybereason
```
Comment (reutshal) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py:325]: move this to the top of the test please.
And you can do:
From Cybereason import malop_processes_command
And then use the command without the prefix

**Review by reutshal — COMMENTED** [review]
@dansterenson Nice!
I don't see a change in the integration's yml file - 
Please make sure that the new output paths are being added to the relevant command outputs.

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py (hunk 2 above)
Comment (dansterenson) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason.py:678]: The previous version which worked was that:
`for i in range(len(PROCESS_INFO)):
            if PROCESS_INFO[i]['type'] == 'filterData':
                output[PROCESS_INFO[i]['header']] = elements[element]['filterData']['groupByValue']`

Therefore I think item was a mistake and in addition, _info_ had the _type_ key and _item_ did not.

**Comment by dansterenson** [comment]
> ure that the

No new outputs were added, the bug was not showing the expected outputs which are written in the .yml file.

**Review by yaakovi — APPROVED** [review]

**Review by reutshal — APPROVED** [review]

**Comment by dansterenson** [comment]
> I suggest adding a UT to the `cybereason-malop-processes` cmd, testing a proper context outputs creation.



> I suggest adding a UT to the `cybereason-malop-processes` cmd, testing a proper context outputs creation.

Done

**Review by bakatzir — APPROVED** [review]
Good job.
